indico/modules/events/management/templates/event_protection.html

```diff
@@ -25,6 +25,11 @@
         </div>
         {{ form_rows(form, fields=form.priv_fields) }}
     {% endcall %}
+    {% if event.type == 'meeting' %}
+        {% call form_fieldset(_("Subcontribution rights"), collapsible=true, initially_collapsed=true) %}
+            {{ form_rows(form, fields=["subcontrib_speakers"]) }}
```

Comment:
we use single quotes :p

and if the field always exists in the form you can use `{{ form_row(form.subcontrib_speakers) }}` directly